src/Microsoft.AspNetCore.Server.Kestrel/Infrastructure/Constants.cs

```diff
@@ -1,14 +1,17 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
+using Microsoft.Extensions.PlatformAbstractions;
+
 namespace Microsoft.AspNetCore.Server.Kestrel.Infrastructure
 {
     internal class Constants
     {
         public const int ListenBacklog = 128;
 
         public const int EOF = -4095;
-        public const int ECONNRESET = -4077;
+        public static readonly int? ECONNRESET = GetECONNRESET();
```

Comment:
This value is read on every call to Connection.OnRead().  Would performance be better if we set this to a sentinel value like int.MinValue if we can't determine the platform?  The perf overhead of nullable int may be small, but I hate to pay an unnecessary cost on every call to OnRead().